### PR TITLE
[映射文档] No. 66

### PR DESCRIPTION
**Pull request by Liyulingyue**
补充gumbel_softmax的文档

https://github.com/PaddlePaddle/PaConvert/pull/118

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+```python
+paddle.nn.functional.gumbel_softmax(x,temperature=1.0,hard=False,axis=-1,name=None)
+```
+
+其中 PyTorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+| PyTorch | PaddlePaddle | 备注                           |
+|--------|--------------|------------------------------|
+| logits | x            | 输入参数                         |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md]: 加一句。仅参数名不同

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+| tau    | temperature  | 0到1的值                        |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md]: 加一句。仅参数名不同

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+| hard   |              | 为True时，将进行one-hot编码，默认为False |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md]: 这个不是对应paddle的hard吗

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+| eps    |              | 计算的误差值                       |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md]: 加一句。暂无转写参数

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+| dim    |axis| 计算的维度                        |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md]: 加一句。仅参数名不同

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+### 转写示例
+
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md]: 这个是需转写哪个参数？目前看只有eps需要写转写示例

https://github.com/PaddlePaddle/docs/blob/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/pytorch_api_mapping_format_cn.md#%E6%A8%A1%E6%9D%BF-6

infoflow 2023-08-29 11-04-10

**Comment by luotao1** [comment]
张师傅，这是66还是67？

**Comment by Liyulingyue** [comment]
> 张师傅，这是66还是67？

应该是66

**Comment by luotao1** [comment]
https://github.com/PaddlePaddle/docs/pull/6146#issue-1869648397

那这里贴的转换规则是67

**Comment by Liyulingyue** [comment]
> [#6146 (comment)](https://github.com/PaddlePaddle/docs/pull/6146#issue-1869648397)
> 
> 那这里贴的转换规则是67

更新了链接的标题~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md (hunk 6 above)
Comment (zhwesky2010) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md]: @Liyulingyue 如果eps可以直接删除的话，就没有需要写转写示例的参数了，无需转写示例

**Comment by zhwesky2010** [comment]
@Liyulingyue 去掉转写示例，这个没有学要转写的参数，另外CI失败了需要处理下

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+# Paddle 写法
+ 
+import paddle
+x = paddle.randn(shape=[20, 32])
+paddle.nn.functional.gumbel_softmax(x)
+
+
+```
```
Comment (Liyulingyue) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.functional.gumbel_softmax.md]:
```suggestion
```

**Comment by zhwesky2010** [comment]
@Liyulingyue 代码风格检查失败了，需要处理下

**Comment by zhwesky2010** [comment]
请问No. 67 torch.nn.functional.pairwise_distance、No. 68 torch.nn.functional.poisson_nll_loss 在哪里，需要同时补一下

**Comment by Liyulingyue** [comment]
> 请问No. 67 torch.nn.functional.pairwise_distance、No. 68 torch.nn.functional.poisson_nll_loss 在哪里，需要同时补一下

最近有点忙，这周末争取补充一下~
